git-server/Cargo.toml

```diff
@@ -28,6 +28,7 @@ tokio = { version = "1.2", features = ["macros", "rt", "rt-multi-thread", "sync"
 tracing = "0.1"
 tracing-subscriber = "0.2"
 warp = { version = "0.3.1", features = ["tls"] }
+radicle-source = { version = "0.2.0", features = ["syntax"] }
```

Comment:
There's no need for the `syntax` feature here, it handles syntax highlighting :)

Reply:
Fixed https://github.com/radicle-dev/radicle-client-services/pull/74/files#diff-7150a884e7aeebff16095bb654e6c671a89cbb34d28864e467368c53ea3bff87R31